quickstart/ia32-generic-qemu.md

```diff
@@ -107,6 +107,151 @@ The number of detected cores is presented during kernel initialization.
 
 ![Image](../_static/images/quickstart/qemu-ia32-generic-smp.png)
 
+## Network setup on ia32-generic-qemu
```

Comment:
We should underline that this setup require busybox applet, so it's required to build the system with `ports` or `all` parameter.